### PR TITLE
SAK-41931 Force forums to use same rubric as gb item

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-41931

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4537339&bid=15272836)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           

Complexity increasing per file
==============================
- rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4537339&bid=15272836)

File: rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js [hunk 1]
```diff
@@ -49,65 +54,56 @@ class SakaiRubricAssociation extends RubricsElement {
...
-        `
-        :
-        html`
-        <span class="sak-banner-info indnt2" style="width: 80%">
-          <sr-lang key="no_rubrics">No rubrics have been created.</sr-lang>
-        </span>
-      `}
       </div>
-      <input name="rbcs-state-details" id="rbcs-state-details" type="hidden" value="${this.stateDetails}">
+      <input name="rbcs-state-details" type="hidden" value="${this.stateDetails}" />
```
[the remark this answers is not in the document]
Comment (adrianfish) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js]: It's hidden, so it shouldn't need a label, I don't think. It just holds the state details, which is a chunk of url encoded json for the server.

[the remark this answers is not in the document]
Comment (ottenhoff) [reply rubrics/tool/src/main/frontend/webcomponents/sakai-rubric-association.js]: agree

**Pull request merged**